### PR TITLE
Moved most debug calls to UberLogger. 

### PR DESCRIPTION
**Pull request by Sommerbo**
Fixed a few typos
Not all `Debug.Log` where replaced due to UberLogger inability to show multiline messages.
SC passed.
And I screwed up commit message.

### CODE REVIEW

File: Assets/Scripts/Models/Temperature.cs [hunk 1]
```diff
@@ -212,7 +212,8 @@ public bool IsWithinTemperatureBounds(float temp)
         }
         else
         {
-            Debug.LogWarning(string.Format("Yep, something is wrong with your temperature: {0}.", temp));
+            // string.format not needed with UberLogger.
```
Comment (mikejbrown) [inline Assets/Scripts/Models/Temperature.cs]: Did not know that. Cool. :smile:

**Comment by mikejbrown** [comment]
I didn't know you could remove `string.Format`, but beyond that I don't see what could possibly be breaking about the diff. I haven't generated the messages that used `string.Format` to check that, but it compiles and runs. :+1:

File: Assets/Scripts/Models/Temperature.cs (hunk 1 above)
Comment (Mizipzor) [inline Assets/Scripts/Models/Temperature.cs]: While correct, this comment shouldnt be committed. :)

File: Assets/Scripts/Models/Temperature.cs [hunk 2]
```diff
@@ -228,9 +229,9 @@ public bool IsWithinThermalDiffusivityBounds(float thermal_diff)
         {
             return true;
         }
-        else
+        else
```
Comment (Mizipzor) [inline Assets/Scripts/Models/Temperature.cs]: Trailing whitespace.

**Comment by Sommerbo** [comment]
Fixed the whitespace

**Pull request merged**